src/algebra/star/big_operators.lean

```diff
@@ -0,0 +1,26 @@
+/-
+Copyright (c) 2021 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import algebra.big_operators.basic
+import algebra.star.basic
+
+/-! # Big-operators lemmas about `star` algebraic operations
+
+Split out from `algebra.star.basic` to remove its `finset` import.
```

Comment:
```suggestion
These results are kept separate from `algebra.star.basic` to avoid it needing to import `finset`.
```
so that this describes the final state and not the process.